apps/omg/lib/omg/state/transaction/payment.ex

```diff
@@ -227,13 +227,17 @@ defimpl OMG.State.Transaction.Protocol, for: OMG.State.Transaction.Payment do
     end
   end
 
-  defp fees_paid(input_amounts_by_currency, output_amounts_by_currency) do
-    input_amounts_by_currency
-    |> Enum.into(%{}, fn {input_currency, input_amount} ->
-      # fee is implicit - it's the difference between funds owned and spend
-      implicit_paid_fee = input_amount - Map.get(output_amounts_by_currency, input_currency, 0)
-      {input_currency, implicit_paid_fee}
-    end)
+  defp input_output_diffs(input_amounts_by_currency, output_amounts_by_currency) do
```

Comment:
similar here, `diffs` is too generic here. If the old name was not ok, then consider just `surplus_by_token`. Also observe that the naming _within_ the function remain's in the "fee world" so the end effect is a bit strange. Maybe easiest is to revert to `fees_paid`, this isn't that bad :thinking: